stripe/functions/package.json

```diff
@@ -20,7 +20,7 @@
     "logs": "firebase functions:log"
   },
   "engines": {
-    "node": "8"
+    "node": "10"
```

Comment:
Let's leave this on 8 for now so all samples in this repo are aligned. There are some changes coming with Node 10 that we need to address across this whole repo. [Background](https://firebase.google.com/support/faq/#expandable-15)

Reply:
These changes mainly seem related to billing, right? Since we're making requests to Stripe we need to enable billing anyway, no?